frontend/packages/dev-console/integration-tests/features/config-maps/config-maps-dev-perspective.feature

```diff
@@ -1,3 +1,4 @@
+@config-maps
```

Comment:
Do we also need `@dev-console` label here?

Reply:
Yes, I have added the tags!